### ISSUE TITLE
Readme images are missing 

### ISSUE THREAD

**Issue by eamonwoortman** [hassio-addons/addon-home-panel#203]
Hi,

Sorry to be "that guy" but the screenshots and banner images in your Readme.md are missing.
Opening them also result in a 404.

Best of luck

**Comment by frenck** [comment]
Yup, seems like upstream the project is working on a new version/overhaul and removed the images/documentation this add-on pulled in.

Awaiting to see what happens upstream at this point.

../Frenck

**Comment by eamonwoortman** [comment]
Right, while 'upstream' is developing the new version, the front page is still lacking the images. 
In favor of not resulting in a stale issue, perhaps we can do a PR for getting the images back.

Here are the images from WayBackMachine (29th March, 2023);

![banner](https://github.com/hassio-addons/addon-home-panel/assets/9072397/3dd702a3-b37e-4b72-918e-52287f1b6fd5)
![screen-main](https://github.com/hassio-addons/addon-home-panel/assets/9072397/cc8907d5-7aba-4ed3-85f1-f11844c197cf)
![screen-configuration-main](https://github.com/hassio-addons/addon-home-panel/assets/9072397/14c593b8-5063-4226-82ab-690850016480)
![purple-space](https://github.com/hassio-addons/addon-home-panel/assets/9072397/a6af707c-b5eb-4e76-9ba5-4c24593c6b2f)

**Comment by frenck** [comment]
Feel free to do so.

**Comment by eamonwoortman** [comment]
Opened the [PR](https://github.com/hassio-addons/addon-home-panel/pull/204), failed the workflow because of missing label.
Not sure how to add it as I seem to be lacking the permission to do so.

**Comment by frenck** [comment]
> Not sure how to add it as I seem to be lacking the permission to do so.

Setting labels on GitHub is not a task a contributor can do.
